pkg/yurtadm/cmd/token/token.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2022 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package token
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"runtime"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+
+	"github.com/openyurtio/openyurt/pkg/yurtadm/constants"
+	"github.com/openyurtio/openyurt/pkg/yurtadm/util"
+	"github.com/openyurtio/openyurt/pkg/yurtadm/util/edgenode"
+)
+
+const (
+	TmpDownloadDir = "/tmp"
+
+	KubeadmUrlFormat      = "https://storage.googleapis.com/kubernetes-release/release/%s/bin/linux/%s/kubeadm"
+	DefaultKubeadmVersion = "v1.22.3"
+
+	MiniumKubeadmVersion = "v1.22.3"
+)
+
+func NewCmdToken(in io.Reader, out io.Writer, outErr io.Writer) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:                "token",
+		Short:              "Run this command in order to manage openyurt joint token",
+		FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if err := CheckAndInstallKubeadm(); err != nil {
+				return err
+			}
+
+			klog.V(2).InfoS("kubeadm command exec", "args", os.Args[2:])
+
+			kubeadmCmd := exec.Command("kubeadm", os.Args[2:]...)
```

Comment:
I am not sure that `os.Args[2:]` should be `os.Args[1:]` or not?

Reply:
It should be `os.Args[1:] `, otherwise the command will become something like `yurtadm token token list`.